scripts/globals/casket_loot.lua

```diff
@@ -14,4046 +14,4046 @@ xi.casket_loot.casketItems =
+            { itemId = xi.item.BARRAGE_TURBINE,                 weight = 240 },
+            { itemId = xi.item.VIVI_VALVE,                      weight = 310 },
+            { itemId = xi.item.DISRUPTOR,                       weight = 290 },
+            { itemId = xi.item.COBALT_CELL,                     weight = 1400 },
+            { itemId = xi.item.RUBICUND_CELL,                   weight = 725 },
+            { itemId = xi.item.XANTHOUS_CELL,                   weight = 1020 },
+            { itemId = xi.item.JADE_CELL,                       weight = 600 },
+            { itemId = xi.item.POUCH_OF_VOIDDUST,               weight = 1750 },
+            { itemId = xi.item.HI_POTION_P3,                    weight = 5000 },
+            { itemId = xi.item.HI_ETHER_P1,                     weight = 4900 },
+            { itemId = xi.item.HI_ELIXIR,                       weight = 1700 },
+            { itemId = xi.item.REMEDY,                          weight = 750 },
+            { itemId = xi.item.HI_RERAISER,                     weight = 2100 },
+            { itemId = xi.item.DRAGON_FRUIT,                    weight = 630 },
+            { itemId = xi.item.FUSION_BOLT_QUIVER,              weight = 1050 },
+            { itemId = xi.item.ERRANT_HAT,                      weight = 80 },
+            { itemId = xi.item.ERRANT_CUFFS,                    weight = 75 },
+            { itemId = xi.item.ERRANT_HOUPPELANDE,              weight = 30 },
+            { itemId = xi.item.VERDUN,                          weight = 110 },
+        },
     },
     [xi.zone.TORAIMARAI_CANAL] =
     {
         regionalItems = { xi.item.BOND_CAPE },
         temps =
         {
-            { xi.item.DAEDALUS_WING,               900 },
-            { xi.item.BOTTLE_OF_CATHOLICON,        500 },
-            { xi.item.PINCH_OF_MANA_POWDER,        300 },
-            { xi.item.FLASK_OF_HEALING_POWDER,     450 },
-            { xi.item.BOTTLE_OF_MONARCHS_DRINK,    320 },
-            { xi.item.BOTTLE_OF_GNOSTICS_DRINK,    650 },
-            { xi.item.BOTTLE_OF_CLERICS_DRINK,     430 },
-            { xi.item.DUSTY_ELIXIR,               1000 },
-            { xi.item.BOTTLE_OF_VICARS_DRINK,      690 },
-            { xi.item.DUSTY_WING,                  250 },
-            { xi.item.LUCID_POTION_I,             2000 },
-            { xi.item.LUCID_POTION_II,            1900 },
-            { xi.item.LUCID_POTION_III,           1200 },
-            { xi.item.FLASK_OF_HEALING_MIST,       730 },
-            { xi.item.FLASK_OF_MANA_MIST,          420 },
-            { xi.item.PAIR_OF_LUCID_WINGS_I,      1010 },
-            { xi.item.TUBE_OF_HEALING_SALVE_I,     875 },
-            { xi.item.TUBE_OF_HEALING_SALVE_II,    210 },
-            { xi.item.TUBE_OF_CLEAR_SALVE_I,       920 },
-            { xi.item.TUBE_OF_CLEAR_SALVE_II,      190 },
-            { xi.item.BOTTLE_OF_STALWARTS_TONIC,   310 },
-            { xi.item.BOTTLE_OF_STALWARTS_GAMBIR,  205 },
-            { xi.i
```

Comment:
My OC syndrome sees the tabkes before, then the tables after and sees an issue.
Please align the weights properly so the numbers remain to the left, like you did in the behemoth suit file down bellow.